src/main/java/uk/gov/hmcts/reform/idam/web/UserController.java

```diff
@@ -117,9 +117,17 @@ public String userActivation(@RequestParam("token") String token, @RequestParam(
         return "useractivation";
     }
 
-    private String buildRegistrationLink(ActivationResult activationResult) {
-        return "/users/selfRegister?redirect_uri=" + activationResult.getRedirectUri() +
-            "&client_id=" + activationResult.getClientId();
+    /**
+     * @should return null if redirecturi or clientid are empty
+     */
+    String buildRegistrationLink(ActivationResult activationResult) {
+        String redirectUri = activationResult.getRedirectUri();
+        String clientId = activationResult.getClientId();
+        if (isEmpty(redirectUri) || isEmpty(clientId)) {
```

Comment:
According to the documentation of `isEmpty`:
> `StringUtils.isEmpty(" ")       = false`

Wouldn't `isBlank` be better in this scenario?